test/CMakeLists.txt

```diff
@@ -231,7 +231,6 @@ function(glmCreateTestGTC NAME)
 	add_test(
 		NAME ${SAMPLE_NAME}
 		COMMAND $<TARGET_FILE:${SAMPLE_NAME}> )
-	target_link_libraries(${SAMPLE_NAME} PRIVATE glm::glm)
```

Comment:
I think this was the right way to link with glm (even if we only have headers). See https://pabloariasal.github.io/2018/02/19/its-time-to-do-cmake-right/#model-dependencies-with-target_link_libraries